hoomd/md/pytest/test_pair.py

```diff
@@ -0,0 +1,751 @@
+import hoomd
+import pytest
+import numpy as np
+import itertools
+from copy import deepcopy
+import json
+from pathlib import Path
+from collections import namedtuple
+
+
+def _assert_equivalent_type_params(type_param1, type_param2):
+    for pair in type_param1:
```

Comment:
I think a docstring about what is expected for argument 1 and 2 would be helpful. I know a GitHub comment was made on this, but having developer documentation on that would be helpful.